bark/benchmark/scenario_dumper.py

```diff
@@ -110,3 +111,56 @@ def write_map(self, config_idx, folder):
       mapfile = params["Scenario"]["Generation"]["ConfigurableScenarioGeneration"]["MapFilename"]
       src = os.path.join("src", "database", mapfile)
       copyfile(src, os.path.join(folder, ntpath.basename(mapfile)))
+
+
+class OpenScenarioDumper(ScenarioDumper):
+  def __init__(self, base_result_folder, benchmark_result):
+    super(OpenScenarioDumper, self).__init__(
+      base_result_folder, benchmark_result)
+  
+  def GetTrajectoryPerAgent(self, world):
+    trajectory_per_agents = {}
+    for (agent_id, agent) in world.agents.items():
+      trajectory_per_agents[agent_id] = []
+      for state_action in agent.history:
+        state = state_action[0]
+        trajectory_per_agents[agent_id].append(
+          [state[0], state[1], state[2], state[3], state[4]])
```

Comment:
maybe use state definition instead of plain index?

Reply:
I admit I just copied and pasted this from the original implementation. Will change that!